packages/replay/src/util/truncateJson/fixJson.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-lines */
+
+import { completeJson } from './completeJson';
+import { evaluateJson } from './evaluateJson';
+
+/**
+ * Takes an incomplete JSON string, and returns a hopefully valid JSON string.
+ * Note that this _can_ fail, so you should check the return value is valid JSON.
```

Comment:
What would cause it to fail? If it fails, what would be the result? Would it be better to throw in that case?

Reply:
Basically, we only know it failed once we tried to `JSON.parse()` it. It can fail in two cases:

1. We incorrectly identified a string as JSON (e.g. think: `{ this is not JSON }`). It will still not be JSON after our fixing, in this case 😅 
2. It _was_ correct JSON, but something went wrong in the implementation. This _should_ not happen, and I know of no case where this would happen, but it is reasonably complex that I wouldn't feel comfortable stating that it will _never_ result in broken JSON.

My first implementation actually try-catched `JSON.parse()` here and threw an error, but the current implementation only has a single JSON.parse in networkUtils, where we try-catch this in a central place, also making it easier to generate the correct warnings for the `_meta` field.